lib/utils/Thing.js

```diff
@@ -32,12 +32,17 @@ export default class Thing {
 		return Thing.visibleThingElements().map(ele => new Thing(ele));
 	}
 
+	static elementFor(element) {
+		return $(element).closest(Thing.thingSelector).get(0) || Thing.visibleThingElements()[0];
```

Comment:
Acknowledging that this is kinda hacky, but I don't think it'll cause any problems since almost everything we interact with is within a `.thing` already